presto-parser/src/main/java/com/facebook/presto/sql/parser/SqlParser.java

```diff
@@ -233,6 +236,7 @@ public void exitNonReserved(SqlBaseParser.NonReservedContext context)
             context.getParent().removeLastChild();
 
             Token token = (Token) context.getChild(0).getPayload();
+            warningConsumer.accept(new ParsingWarning("nonReserved word used", token.getLine(), token.getCharPositionInLine()));
```

Comment:
Why not `Non-reserved`?

Reply:
Updated.
Should I also update the assertion on line 232 to match (i.e. change "nonReserved" to "Non-reserved" there as well)?